src/style.css

```diff
@@ -0,0 +1,21 @@
+
+.red-block {
+  background-color: red;
+}
+
+.green-block {
+  background-color: green;
+}
+
+.blue-block {
+  background-color: blue;
+}
+
+body {
```

Comment:
better to write common style on the top of the document.
move `body` to the top.